src/nidcpower/metadata/functions_addon.py

```diff
@@ -148,8 +155,46 @@
             'description': 'Returns the number of channels.',
         },
     },
+    # Public function that wraps self_test and will raise on self test failure
+    'self_test_wrapper': {
```

Comment:
I truly feel this should be Fancy Self Test.

Reply:
Changed